File: source/Oddities/oddities.java
import java.util.Scanner;

public class oddities {

    public static void main(String[] args) {
      
        Scanner in = new Scanner(System.in);

        
        int a[] = new int[20];
        
        int n = in.nextInt();
        
        for(int i = 0; i < n; i++){
            a[i] = in.nextInt();
        }
        
        for (int i = 0; i < n; i++) {
            if(a[i] % 2 == 0){
                System.out.println(a[i] + " is even ");
            }else{
                System.out.println(a[i] + " is odd ");
            }
        }
        
    
    }

}

File: source/Oddities/oddities.cpp
#include <iostream>

using namespace std;

int main() {
	int loop, num;
	cin >> loop;
	for (int i=0; i<loop; i++) {
		cin >> num;
		if(num%2 == 0) {
			cout << num << " is even \n";
		} else {
			cout << num << " is odd \n";
		}
	}
	return 0;
}
